### ISSUE TITLE
Grouped Columns Don't Render Correctly when downloading as a PDF

### ISSUE THREAD

**Issue by daylanv** [olifolkerd/tabulator#3310]
When I render my table in the browser, everything looks correct. My first column isn't grouped, my next three columns are grouped (two with two subgroups each, and the final one with three subgroups). When I download it to a PDF, the generated autotable doesn't match - the headers don't line up with the cells.

**Tabulator Info**
Version 4.9.3

**Working Example**
https://codepen.io/daylandaylan/pen/ZEebjyp?editors=1111

**To Reproduce**
You can see the table is rendered correctly in the HTML, but if you click the download button at the bottom, the generated pdf's totals header doesn't stretch to fill the page - and the "products" header is aligned above the wrong columns

**Expected behavior**
The grouped column headers should be centered above their subheaders.

**Screenshots**
Correct layout:
![image](https://user-images.githubusercontent.com/50340007/117841137-a10c1f80-b231-11eb-8793-15b90dce882c.png)
Layout being generated:
![image](https://user-images.githubusercontent.com/50340007/117841161-a9645a80-b231-11eb-85a1-4c69d8e1d3b5.png)

**Desktop (please complete the following information):**
Chrome 90, windows.

**Additional context**
Outlining it, I see empty cells causing a partial offset. The colspawn being generated between the header and the rest doesn't align.
![image](https://user-images.githubusercontent.com/50340007/117843682-e29dca00-b233-11eb-9ebb-7c555f28a137.png)

The example code for column grouping (http://tabulator.info/examples/4.9?#column-groups) has the same issue

**Comment by olifolkerd** [comment]
Are u using the versions of the autotable library recomended on the Tabulator website?

**Comment by daylanv** [comment]
> Are u using the versions of the autotable library recomended on the Tabulator website?

Yeah using the two versions listed under here:
http://tabulator.info/docs/4.9/download#pdf

I also tried using the more recent versions just to confirm that it wasn't a bug with jsPDF/autotable itself.


I managed to get it working by removing the "else" statement in the parseRow function (i.e. not pushing an empty string into the rowData array). I'm not sure what other functionality this line provides though.
![image](https://user-images.githubusercontent.com/50340007/124502324-1ef32f80-dd78-11eb-80f4-3f01cf3120c9.png)
